components/dashboards-web-component/src/designer/DashboardDesigner.jsx

```diff
@@ -1,31 +1,33 @@
 /*
- *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
I don't think we should update the year of existing files

Reply:
fixed with 6d52b52cc22ff133f368451e718a4f70913fc0ee